src/cmd/info.rs

```diff
@@ -80,14 +80,20 @@ fn print_wallet(wallet: &Wallet, account: &Account, format: OutputFormat) -> Res
             print_table(&table, None)
         }
         OutputFormat::Json => {
-            let table = json!({
+            let json = json!({
                 "sharded": wallet.is_sharded(),
                 "network": wallet.public_key.key_tag().network.to_string(),
                 "type": wallet.public_key.key_tag().key_type.to_string(),
                 "pwhash": wallet.pwhash().to_string(),
                 "account": account,
+                "solana_address": if let Ok(solana_key) = solana_sdk::pubkey::Pubkey::try_from(wallet.public_key.clone())
```

Comment:
I think you can replace all of this if statement with 

```
solana_sdk::pubkey::Pubkey::try_from(wallet.public_key.clone()).ok()
```

Reply:
My goal is to return `null` for the JSON value if it can't covert. Will this do that?

Reply:
(it turns out to be remarkably hard for me to get an `ecc_compact` key into the wallet to test with).

Reply:
Indeed it will: https://doc.rust-lang.org/std/result/enum.Result.html#method.ok

Reply:
Wonderful. Adopted. Just waiting for GitHub to see the push.